across_data_ingestion/tasks/schedules/swift/util.py

```diff
@@ -0,0 +1,404 @@
+from typing import Any, Callable
+
+import structlog
+from astropy.time import Time  # type: ignore[import-untyped]
+from swifttools import swift_too  # type: ignore
+from swifttools.swift_too.swift_obsquery import Swift_AFST_Entry  # type: ignore
+from swifttools.swift_too.swift_planquery import PPSTEntry  # type: ignore
+from swifttools.swift_too.swift_uvot import UVOTModeEntry  # type: ignore
+
+from ....util.across_server import client, sdk
+
+logger: structlog.stdlib.BoundLogger = structlog.get_logger()
+
+SWIFT_XRT_BANDPASS = sdk.EnergyBandpass(
+    filter_name="Swift XRT",
+    min=0.3,
+    max=10.0,
+    unit=sdk.EnergyUnit.KEV,
+)
+
+SWIFT_BAT_BANDPASS = sdk.EnergyBandpass(
+    filter_name="Swift BAT",
+    min=15.0,
+    max=150.0,
+    unit=sdk.EnergyUnit.KEV,
+)
+
+SWIFT_UVOT_BANDPASS_DICT = {
```

Comment:
while we are refactoring some of this, we should move these constants into a constants.py file for swift